knora/dsplib/utils/xml_upload.py

```diff
@@ -5,14 +5,12 @@
 import json
```

Comment:
Why did I change this file?
 - I moved `_try_network_action()` to `shared_methods.py` 
 - I changed the behaviour of `_try_network_action()`: If the connectivity issues remain, it won't return None anymore, but it will escalate the error that it received from the method that failed. For this reasons, I have to embed all usages of `_try_network_action()` into a try-except block